template/composer.json

```diff
@@ -1,10 +1,9 @@
 {
     "name": "zendframework/<package_name>",
     "description": "<package_description, the same like the repository description>",
-    "homepage": "https://docs.zendframework.com/<package_name>/",
```

Comment:
This should still be present, as it allows that information to be displayed when you run `composer show <packagename>`.

Reply:
but it should be link to the docs or rather to the github repo ?

Reply:
Never mind. I misunderstood the commit as indicating the information was in the `SUPPORT.md` file, when you were referencing the `support` section of the `composer.json`.